package.json

```diff
@@ -6,32 +6,37 @@
   "license": "MIT",
   "repository": "radi-js/radi",
   "author": "Marcis Bergmanis <marcisbergmanis@gmail.com>",
-  "keywords": [
-    "radi",
-    "radijs",
-    "radi.js",
-    "vdom"
-  ],
+  "keywords": ["radi", "radijs", "radi.js", "vdom"],
   "scripts": {
     "pack": "webpack --config ./.webpack/dev.js",
     "build": "npm run bundle",
     "bundle": "rollup -i src/index.js -o dist/radi.js -m -f umd -n radi",
     "lint": "eslint src/**/*.js",
-    "test": "jest"
+    "test": "jest",
+    "precommit": "pretty-quick --staged"
   },
   "devDependencies": {
     "babel-preset-env": "^1.6.1",
     "eslint": "^4.18.2",
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "^2.9.0",
     "http-server": "^0.11.1",
+    "husky": "^0.14.3",
     "jest": "^22.4.2",
     "jsdom": "^11.5.1",
+    "prettier": "^1.11.1",
+    "pretty-quick": "^1.4.1",
     "sinon": "^4.4.2",
     "webpack": "^4.1.1",
     "webpack-cli": "^2.0.10"
   },
-  "prettier": {},
+  "babel": {},
   "jest": {},
-  "babel": {}
+  "prettier": {
+    "tabWidth": 2,
+    "trailingComma": "none",
```

Comment:
I'd argue for a trailing comma in objects when multiline. It:

- is more convenient to add more things to object lists
- makes for cleaner diffs (when you add a line, the previous one is not indicated as changed). Makes reviewing just a tad faster

But let's see what the rest thinks as well :-)

Reply:
Agree! I use trailing commas for same reasons.
I'd like to have them in Radi source as well

Reply:
What I love about prettier is that the point of it is for me not to be concerned with it. So I am good with whatever you guys like...

Except the indentation. I'd like to debate that if it comes up, haha.

@ambewas As stated, I'm good with either one, no problem. But I want to know how often you plan to be manually creating data structures, and also how much effort it really requires to add a comma? I believe more mental energy went into your comment than will go into adding commas to `{}[]` in 2018.

Reply:
Ehh, it's not a lot of effort, sure. But can't hurt to eliminate an extra keystroke here and there = ). 

I'm mostly fond of it because of working with PropTypes and the like. Just duplicating a line and not having to add/remove the comma is easier :)